### PR TITLE
Dependencies added

### PR DESCRIPTION
**Pull request by govindraj111**
# Associated Issue: #16 

## Key Change Summary

Note: Dependency for Player Particles could not be added.
Dependencies added: ProtocolLib, Vault API, Libs Disguises, WorldGuard, WorldEdit.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -120,5 +128,34 @@
             <version>${junit.jupiter.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.azbh111</groupId>
```
[the remark this answers is not in the document]
Comment (sirNikolai) [reply pom.xml:132]: This should be the dmulloy one

**Review by sirNikolai — CHANGES_REQUESTED** [review]
Thanks for the contribution.
My bad for not specifying but should be valid for version `1.16.2` at least.
Some minor bits and pieces that need changing here, comments mention most of them.

The program is also failing to compile because of libs disguises not being included (as mentioned in a different comment). This should be imported through maven and not treated as a system provided resource.

**Comment by govindraj111** [comment]
@sirNikolai will do that!

**Comment by govindraj111** [comment]
@sirNikolai Please review the changes!

**Comment by sirNikolai** [comment]
This required fixes. It got merged in with more changes. In future please run locally to make sure this works (at least compiles) bore finalizing PRs.

**Comment by govindraj111** [comment]
:+1:

**Comment by govindraj111** [comment]
@sirNikolai would request you to label this PR as **hacktoberfest-accepted**

**Comment by govindraj111** [comment]
>> **Hey, an update from hactoberfest states that the PR should be labeled as ‘hacktoberfest-accepted’ by a maintainer.**
Screenshot from 2020-10-16 08-04-27
Screenshot from 2020-10-16 08-06-57

>> **Please label this PR as hacktoberfest-accepted**
hacktoberfest-label-add-gif-min

**Pull request merged**